### ISSUE TITLE
Ask about the training time of the server used in this paper

### ISSUE THREAD

**Issue by LLLLLearn** [iwangjian/TRIP#1]
Dear author
I would like to know the total training time for the models. This information is crucial for me to allocate resources appropriately and estimate the time required for my experiments.

Thank you very much for taking the time to read my issue. I look forward to your response and hope to have the opportunity to learn more from your work.

**Comment by iwangjian** [comment]
Hi, thanks for your interest in our work. The experiments of this work were conducted two years ago and the logs regarding the accurate total training time were missing. First of all, our experiments were conducted with a single NVIDIA 3090 GPU. Approximately, training a dialogue planning model with 10 epochs on our datasets costs about 10~12 hours (maybe more or less, I cannot remember clearly). For dialogue generation model training, the *prompt-based* method is quite fast as it is simply fine-tuning GPT-2, while the *plan-controlled* method is a little time-consuming as it requires gradient computation during generation.

**Comment by LLLLLearn** [comment]
Thank you very much for your reply.
